build/test/integration/distribution_test.js

```diff
@@ -3,7 +3,7 @@
 /* jshint maxlen: 130 */
 
 var rmrf = require('rimraf').sync;
-var AdmZip = require('adm-zip');
+// var AdmZip = require('adm-zip');
```

Comment:
Might as well just delete the line. Guessing you were probably going to do that. :)